ground/src/main/java/com/google/android/ground/persistence/local/room/converter/ConverterExt.kt

```diff
@@ -85,21 +85,7 @@ fun BaseMap.toLocalDataStoreObject(surveyId: String) =
 
 fun BaseMapEntity.toModelObject() = BaseMap(url = URL(url), type = type.toModelObject())
 
-fun GeometryEntity.toModelObject() =
-  when (geometryType) {
-    GeometryType.POINT.name -> this.toPointModel()
-    GeometryType.POLYGON.name -> this.toPolygonModel()
-    else -> null
-  }
-
-private fun GeometryEntity.toPointModel(): Geometry? = this.location?.toPoint()
-
-private fun GeometryEntity.toPolygonModel(): Geometry {
-  val shell = LinearRing(parseVertices(this.vertices).map { it.coordinate })
-  val holes = parseHoles(this.holes).map { LinearRing(it.map(Point::coordinate)) }
-
-  return Polygon(shell, holes)
-}
+fun Geometry.toModelObject() = GeometryWrapper(this)
```

Comment:
Did you mean `Geometry.toLocalDataStoreObject()`?

Reply:
Oops yea, good catch!